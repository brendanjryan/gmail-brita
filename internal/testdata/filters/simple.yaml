emails:
  - me@example.com

filters:
  - name: Simple Test Filter
    conditions:
      has:
        - test:condition
    actions:
      label: test-label
      archive_unless_directed:
        mark_read: false 
