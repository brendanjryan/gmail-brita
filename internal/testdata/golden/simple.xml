<?xml version="1.0" encoding="UTF-8"?>
<feed xmlns="http://www.w3.org/2005/Atom" xmlns:apps="http://schemas.google.com/apps/2006">
  <title>Mail Filters</title>
  <entry>
    <category term="filter"></category>
    <title>Mail Filter</title>
    <apps:property name="hasTheWord" value="test:condition"/>
    <apps:property name="label" value="test-label"/>
    <apps:property name="shouldArchive" value="true"/>
    <apps:property name="shouldMarkAsRead" value="false"/>
  </entry>
</feed> 
